patient-management/patient-management-api/src/main/java/org/leanhis/patientmanagement/PatientService.java

```diff
@@ -0,0 +1,17 @@
+package org.leanhis.patientmanagement;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface PatientService {
+
+    Patient create(Patient patient);
+
+    List<Patient> findBy(String nameOrId);
```

Comment:
Search for patients in the by their name or patient number.

`findByNameOrNumber`